pkgs/development/python-modules/pyspread/default.nix

```diff
@@ -1,56 +1,67 @@
-{ buildPythonPackage
-, fetchPypi
-, isPy3k
-, stdenv
-, numpy
-, wxPython
-, matplotlib
-, pycairo
-, python-gnupg
-, xlrd
-, xlwt
-, jedi
-, pyenchant
-, basemap
-, pygtk
+{ lib
+, pythonFull
+, pyspread-unwrapped
 , makeDesktopItem
+, extraPythonDependencies ? []
+# Former version: [ matplotlib pycairo python-gnupg xlrd xlwt pyenchant basemap ]
+# requirements.txt optional: [ matplotlib pyenchant pip ]
```

Comment:
generally people just add this to their environment:
```
python3.withPackages(ps: with ps; [
  pyspread
  matplotlib
```suggestion
```
  pyenchant
  pip
])
```

Reply:
I have it removed, and would write a NixOS module to make it easier to add custom dependencies if I have time.